CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [9.2.1] - 2022-05-05
+
+# Adds
+
+- Support for h3 freamwork.
```

Comment:
this should go under the unreleased section

Reply:
Ah my bad. I'll fix it

Reply:
wait it is under unreleased

Reply:
It seems to be under `## [9.2.1] - 2022-05-05`

Reply:
AHHHH mb mb

Reply:
there